sbk-gem/javadoc/io/gem/api/SshSession.html

```
@@ -91,7 +91,7 @@ <h1 title="Class SshSession" class="title">Class SshSession</h1>
 <span class="extends-implements">extends <a href="https://docs.oracle.com/en/java/javase/25/docs/api/java.base/java/lang/Object.html" title="class or interface in java.lang" class="external-link">Object</a></span></div>
 <div class="block">Lifecycle wrapper around an SSH client/session for a single connection.
 
-<p>Encapsulates connect, command execution, SCP directory copy, and graceful shutdown,
+<p>Encapsulates connect, command execution, SCP file copy, and graceful shutdown,
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 94.

(tag-pair)